File: Python/program7.py
# program7.py
def addition_two_numbers(i_no1, i_no2):
    i_sum = i_no1 + i_no2  # Business Logic
    return i_sum

def main():
    print("Enter first number : ")
    i_value1 = int(input())

    print("Enter second number : ")
    i_value2 = int(input())
    
    i_ret = addition_two_numbers(i_value1, i_value2)
    print(f"Addition is : {i_ret}")

if __name__ == "__main__":
    main()

File: Java/Program7.java
import java.util.Scanner;

public class Program7 {

    public static int additionTwoNumbers(int no1, int no2) {
        return no1 + no2;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int value1, value2;

        System.out.print("Enter first number: ");
        value1 = sc.nextInt();

        System.out.print("Enter second number: ");
        value2 = sc.nextInt();

        int ret = additionTwoNumbers(value1, value2);

        System.out.println("Addition is: " + ret);
    }
}